### DISCUSSION TITLE
branch-2.4.3

### DISCUSSION THREAD

**Discussion by ianthomas23** [bokeh/bokeh#12125] [General]
I've created a `branch-2.4.3` off `branch-2.4` for the upcoming minor release. So far it contains the following cherry-picked PRs:

1. #12113
2. #12034
3. #12084
4. #12121
5. #11906 (code changes but not removal of baseline images)

As far as testing goes, all bokehjs tests pass in a new Python 3.10 conda environment on Ubuntu 20.10 using (old) chromium 95.0.4638.69 via snap. There are lots of problems with Python testing as I don't have a fully working selenium/chromedriver combination (but I haven't spend much time on this).

**Comment by bryevdv** [comment]
FYI Here is an example backport PR

* https://github.com/bokeh/bokeh/pull/11805

Pretty much the info you have above. I would say let's go ahead and start a PR and link to this discussion, and see how the tests fare in CI

**Comment by bryevdv** [comment]
I've gone ahead and created the 2.4.3 milestone:

https://github.com/bokeh/bokeh/milestone/73

**Reply by ianthomas23** [reply]
@bryevdv Can you clarify the intended use of the 2.4.3 milestone? Is it for already-merged PRs to indicate that they need to be backported, or open issues/PRs to indicate they need to go into 2.4.3 as well as 3.0, or both of the above? Thanks!

**Reply by bryevdv** [reply]
@ianthomas23 At the end of the day, the milestones are an exclusive record of what went into a given release. The milestones drive the automated changelog generation. Every issue and/or PR that we are backporting needs to be moved out of the 3.0 milestone and into the 2.4.3 milestone. 

As for the process, I don't really care :) It just needs to be correct by the time we hit the release button.

**Reply by bryevdv** [reply]
I will say, this is why I find listing the individual PRs in the backport PR to be valuable. Before release, we will want to go through and double check everything. Having a list in the backport PR makes it easier to do that verification.

**Reply by ianthomas23** [reply]
@bryevdv Thanks for clarifying, that makes complete sense.
